packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -41,7 +41,7 @@ enum CircleVisbility {
 }
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30192&commitId=434dafe26e235699a0c732727049c16c0c15b3b7&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-VerticalBarChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels - Dark Mode.default.chromium.png|7676| Changed|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels.default.chromium.png|28902| Changed|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels - RTL.default.chromium.png|28892| Changed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends - Dark Mode.default.chromium.png|24227| Changed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends.default.chromium.png|31461| Changed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends - RTL.default.chromium.png|31467| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>